src/index.html

```diff
@@ -13,31 +13,22 @@ <h1>Leaderboard</h1>
         <section class="scores">
           <header>
             <h2>Recent scores</h2>
-            <button type="button">Refresh</button>
+            <button type="button" id="refresh">Refresh</button>
           </header>
-          <div class="scores__container">
-            <p>Name: 100</p>
-            <p>Name: 20</p>
-            <p>Name: 50</p>
-            <p>Name: 78</p>
-            <p>Name: 125</p>
-            <p>Name: 77</p>
-            <p>Name: 42</p>
-          </div>
+          <div class="scores__container"></div>
         </section>
         <section class="form__components">
           <header>
             <h2>Add your score</h2>
-            <form>
-              <input type="text" placeholder="Your name" required />
+            <form class="form">
+              <input type="text" id="name" placeholder="Your name" required />
               <input
```

Comment:
Good Job!!
I will suggest you make your input more controlled to accept only positive integers